pkg/tecdsa/signing/gen/pb/message.proto

```diff
@@ -44,4 +44,10 @@ message TSSRoundSixMessage {
     uint32 senderID = 1;
     bytes payload = 2;
     string sessionID = 3;
+}
+
+message TSSRoundSevenMessage {
+    uint32 senderID = 1;
+    bytes payload = 2;
```

Comment:
Let's rename it to `broadcastPayload` as noted during the review of other PRs from the signing series.

Reply:
Addressed by https://github.com/keep-network/keep-core/pull/3271